### PR TITLE
chore: ensure we do not set config to undefined onPreBuild

### PR DESCRIPTION
**Pull request by lemusthelroy**
Release-As: 0.0.25

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -10,6 +10,7 @@ export const onPreBuild = ({
   const emailsDirectory = process.env.NETLIFY_EMAILS_DIRECTORY ?? "./emails";
 
   netlifyConfig.functions.emails = {
+    ...netlifyConfig.functions.emails,
     included_files: [`${emailsDirectory}/**`],
```
Comment (estephinson) [inline src/index.ts:14]: Need to spread the included files too my friend

**Review by estephinson — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (estephinson) [reply src/index.ts:14]: unless this is something that will never need to be modified by the user?

Comment (lemusthelroy) [reply src/index.ts:14]: Yeah I don't think I would want users modifying that tbh

Comment (lemusthelroy) [reply src/index.ts:14]: hmmmm but maybe there are email assets like images

**Review by estephinson — APPROVED** [review]

**Pull request merged**